config/config/common-config.yaml

```diff
@@ -818,6 +818,12 @@ opa:
     tags:
       - latest
 
+  ## Prevent the creation of load balancer services when unsupported
+  rejectLoadBalancerService:
+    enabled: false
```

Comment:
Could you change this to `set-me` and add an init fuction that sets it based on the cloud provider? Something like this function https://github.com/elastisys/compliantkubernetes-apps/blob/main/bin/init.bash#L267

AFAIK safespring, upcloud, exoscale, and baremetal should be `enabled: true` and `false` for the other ones.